src/lib/slp/dialogs/service_selection.rb

```diff
@@ -0,0 +1,163 @@
+# ------------------------------------------------------------------------------
+# Copyright (c) 2016 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify it under
+# the terms of version 2 of the GNU General Public License as published by the
+# Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+#
+# ------------------------------------------------------------------------------
+
+require "yast"
+require "ui/dialog"
+
+module Yast
+  module Dialogs
+    # This class implements a services selection dialog.
```

Comment:
It should be `Y2Slp` IIRC. But as we don't have a final agreement yet, I named in the *old-fashing*.